src/transformers/onnx/config.py

```diff
@@ -295,6 +314,10 @@ def generate_dummy_inputs(
             )
             # Generate dummy inputs according to compute batch and sequence
             dummy_input = [" ".join([preprocessor.unk_token]) * seq_length] * batch_size
+            # Generate dummy bounding boxes if needed by the preprocessor e.g. for LayoutLMv2
+            if return_bbox is True:
```

Comment:
If the `return_bbox` is True then it means the preprocessor needs bounding boxes and it returns the appropriate dictionary with dummy_bbox.